### PR TITLE
Create Blog Report for TMVA

### PR DESCRIPTION
**Pull request by Harshalzzzzzzz**

### CODE REVIEW

**Review by agheata — CHANGES_REQUESTED** [review]
Please fix the errors reported by the link checker. Also provide the missing link for PR 2 and ask your mentor(s) to review this.

**Comment by agheata** [comment]
@Harshalzzzzzzz can you please make the requested changes to be able to merge this?

**Comment by Harshalzzzzzzz** [comment]
> @Harshalzzzzzzz can you please make the requested changes to be able to merge this?

@agheata I have updated the link
I will ask my mentors to review this
Thank you !

**Review by lmoneta — APPROVED** [review]
It is fine for me

File: _gsocblogs/2022/blog_TMVADevelopments_HarshalShende.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+)
+```
+
+### [PR 2](https://github.com/root-project/root/compare/master...Harshalzzzzzzz:root:hist-pythonizations) 
+- Added pythonizations for TGraph and Hist methods, functions to create Histograms objects (TH1, TH2, TH3) from NumPy arrays and TGraph's objects from NumPy.
+
+
+## Relevant Links
+
+ - [Project Proposal]([https://docs.google.com/presentation/d/15zMAqtXGF_JnOSiL8Pe5A9m1hezotmRIGW_Dgp9ZJHQ/edit#slide=id.p](https://docs.google.com/document/d/1Ss_nWnIUPxGdg4l5dX3RK5670iRZjSujFoIwUAkM4FA/edit?usp=sharing))
```
Comment (agheata) [inline _gsocblogs/2022/blog_TMVADevelopments_HarshalShende.md:66]: This link is wrong, keep the actual link to the document with the format: `[Project Proposal](link)`

**Review by agheata — CHANGES_REQUESTED** [review]

**Comment by agheata** [comment]
> I have updated the link

There is still a wrongly formatted link, see above

**Review by agheata — APPROVED** [review]

**Pull request merged**